openfecwebapp/tests/selenium/base_test_class.py

```diff
@@ -10,6 +11,10 @@
 
 from openfecwebapp.sauce import SauceClient
 
+# Silence Selenium logs
+remote_logger = logging.getLogger('selenium.webdriver.remote.remote_connection')
+remote_logger.setLevel(logging.WARN)
```

Comment:
For this logger, we'll only see messaged logged at `WARN`, `ERROR`, or `CRITICAL`. All other loggers are unaffected.